homeassistant/components/watchdog_file_watcher.py

```diff
@@ -0,0 +1,75 @@
+"""
+Component for monitoring activity on a folder.
```

Comment:
The name of this component is weird.

 - it's watching a folder
 - but it's called watchdog_file_watcher

Watchdog is the requirement that is used. Probably just `folder_watcher` would be an ok name ?

Reply:
Also don't forget to specify a `REQUIREMENTS` constant with watchdog in it

Reply:
Yes I'm happy with `folder_watcher`